hbase-server/src/main/java/org/apache/hadoop/hbase/master/webapp/RegionReplicaInfo.java

```diff
@@ -17,14 +17,22 @@
  */
 package org.apache.hadoop.hbase.master.webapp;
 
+import static org.apache.hbase.thirdparty.org.apache.commons.collections4.ListUtils.emptyIfNull;
```

Comment:
A few unused imports to cleanup.

Reply:
Thanks! I have fixed this.

p.s. For a PR to Hadoop, I usually check the "checkstyle" report by Yetus and update the patch accordingly. I see the Yetus for HBase QA does not report "checkstyle" warnings in the comment table. I found there are report links in the "Console output" though. I'll follow there next time. :)

Reply:
Due to complications around multi-jdk support, we have Yetus run in 3 different invocations. Looks like the checkstyle check gave you a +1.